docs/guides/supported_backends.md

```diff
@@ -0,0 +1,51 @@
+# Supported Backends 
+
+
+GuideLLM requires an OpenAI-compatible server to run evaluations. [vLLM](https://github.com/vllm-project/vllm) is recommended for this purpose; however, GuideLLM is compatible with many backend inference servers such as TGI, llama.cpp, and DeepSparse. 
+
+## OpenAI/HTTP Backends
+
+### Text Generation Inference 
+[Text Generation Inference](https://github.com/huggingface/text-generation-inference) can be used with GuideLLM. To start a TGI server with a Llama 3.1 8B using docker, run the following command:
```

Comment:
I believe markdown specifies a CR between section headers (or any control block) and new text.